src/include/server/mir/server_configuration.h

```diff
@@ -114,6 +115,8 @@ class ServerConfiguration
 
     virtual auto the_decoration_strategy() -> std::shared_ptr<DecorationStrategy> = 0;
     virtual void set_the_decoration_strategy(std::shared_ptr<DecorationStrategy> strategy) = 0;
+
+    virtual void set_the_decoration_manager(std::shared_ptr<shell::decoration::Manager> manager) = 0;
```

Comment:
This is a pointless API. There's no way for a caller to pass a `shell::decoration::Manager` as that interface isn't available to implement

Reply:
This is more of a leftover from when I was experimenting with decorations. I'll push a commit to "expose" that interface (apologies in advance if I do it wrong)

Reply:
Should be handled now